Makefile

```diff
@@ -38,15 +38,15 @@ override CFLAGS += $(shell $(PKG_CONFIG) --cflags glib-2.0) -DVERSION=\"$(VERSIO
 # "pkg-config --exists" will error if the package doesn't exist. Make can only compare
 # output of commands, so the echo commands are to allow pkg-config to error out, make to catch it,
 # and allow the compilation to complete.
-ifeq ($(shell $(PKG_CONFIG) --exists libsystemd-journal && echo "0" || echo "1"), 0)
+ifeq ($(shell $(PKG_CONFIG) --exists libsystemd-journal && echo "0"), 0)
 	override LIBS += $(shell $(PKG_CONFIG) --libs libsystemd-journal)
-	override CFLAGS += $(shell $(PKG_CONFIG) --cflags libsystemd-journal) -D USE_JOURNALD=0
-else ifeq ($(shell $(PKG_CONFIG) --exists libsystemd && echo "0" || echo "1"), 0)
+	override CFLAGS += $(shell $(PKG_CONFIG) --cflags libsystemd-journal) -D USE_JOURNALD=1
+else ifeq ($(shell $(PKG_CONFIG) --exists libsystemd && echo "0"), 0)
 	override LIBS += $(shell $(PKG_CONFIG) --libs libsystemd)
-	override CFLAGS += $(shell $(PKG_CONFIG) --cflags libsystemd) -D USE_JOURNALD=0
+	override CFLAGS += $(shell $(PKG_CONFIG) --cflags libsystemd) -D USE_JOURNALD=1
 endif
 
-ifeq ($(shell $(PKG_CONFIG) --atleast-version 2.5.0 libseccomp && echo "0" || echo "1"), 0)
+ifeq ($(shell $(PKG_CONFIG) --atleast-version 2.5.0 libseccomp && grep -rq "seccomp_notif_sizes" /usr/include && echo "0"), 0)
```

Comment:
this could be expensive, and detect the string in other unrelated header files.

What do you think about replacing it with `printf "#include <linux/seccomp.h>\nvoid main(){struct seccomp_notif_sizes s;}" | cc -x c - -o /dev/null` ?

Reply:
SGTM

Reply:
updated as suggested!